make/helm.mk

```diff
@@ -11,3 +13,21 @@ lint-and-install-chart: ## Lints and installs helm chart
 lint-and-install-chart:
 	ct lint-and-install --config charts/ct-config.yaml
 	ct lint-and-install --config charts/ct-config.yaml --upgrade
+
+.PHONY: schema-chart
+schema-chart: ## Updates helm values JSON schema
+schema-chart: helm-values-schema-plugin
+	HELM_PLUGINS=$(HELM_PLUGINS) helm schema \
+	--input charts/cluster-api-runtime-extensions-nutanix/values.yaml \
+	--output charts/cluster-api-runtime-extensions-nutanix/values.schema.json
+
+.PHONY: helm-values-schema-plugin
+helm-values-schema-plugin: ## Installs helm-values-schema plugin
+helm-values-schema-plugin:
+	# Only install if plugin not yet installed
+	HELM_PLUGINS=$(HELM_PLUGINS) \
+	helm plugin list | grep --silent "^schema\s" \
+		|| \
+	HELM_PLUGINS=$(HELM_PLUGINS) \
+	helm plugin install \
```

Comment:
can we pin the version using `--version v1.4.1`

Reply:
I've pushed a PR to this PR branch to use devbox (and nix) to package up this helm plugin (https://github.com/dlipovetsky/capi-runtime-extensions/pull/2).

Reply:
@supershal Good point. It's pinned now.